include/godot_cpp/classes/wrapped.hpp

```diff
@@ -95,6 +108,8 @@ class Wrapped {
 	Wrapped(GodotObject *p_godot_object);
 	virtual ~Wrapped() {}
 
+	static const StringName *_get_extension_class_name(); // This is needed to retrieve the class name before the godot object has its _extension and _extension_instance members assigned.
```

Comment:
Could you move this back up to where this method was declared before (just before `void _notification(int p_what) {}`)? It doesn't matter functionally, but minimizing changes may help with preventing merge conflicts between PRs.